.github/workflows/fuzzing-run.yml

```diff
@@ -0,0 +1,61 @@
+name: Fuzzer execution
+
+on:
+  workflow_call:
+    inputs:
+      target:
+        required: true
+        type: string
+      arch:
+        required: true
+        type: string
+      package-type:
+        required: true
+        type: string
+      timeout-seconds:
+        description: The maximum time in seconds to run the fuzzer.
+        required: false
+        type: number
+        default: 600
+
+jobs:
+  package:
+    uses: ./.github/workflows/package-build.yml
+    with:
+      target: ${{ inputs.target }}
+      arch: ${{ inputs.arch }}
+      artifact: ${{ inputs.target }}-fuzzer
+      package-type: ${{ inputs.package-type }}
+      build-fuzzers: true
+
+  fuzzer-execution:
+    needs: package
+    runs-on: [self-hosted, 1ES.Pool=e2e-pool, '1ES.ImageOverride=${{ inputs.target }}']
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/download-artifact@v4
+        id: download
+        with:
+          name: ${{ inputs.target }}-fuzzer
+
+      - name: Install prerequisites
+        run: |
+          sudo apt update
+          sudo apt install -y clang
```

Comment:
Unnecessary since we have it in the container now

Reply:
This has been discussed under https://github.com/Azure/azure-osconfig/pull/792 which got closed after rebasing with main. Needed at runtime, let's keep it now.